docs/chaos-engineering/User Guides/connect-chaos-infrastructures.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Connect Chaos Infrastructures
+---
+
+Chaos infrastructure is a service that runs in your target environment and aids HCE in accessing and injecting chaos at cloud-native scale. There are different types of chaos infrastructures based on the target environments, such as Kubernetes, Linux VMs, AWS cloud, VMware, etc. Therefore, these chaos infrastructures might install as a Kubernetes service or a Linux daemon and so on, based on their type. All the chaos infrastructure services adhere to the principle of least privilege where they execute only with the minimum number of permissions required.
+
+## Create Environment
+To add a chaos infrastructure for your target environment, first we need to create a new environment as part of which the chaos infrastructures will exist. Head to the **Environments** page, which lists all the environments added as part of your project. Choose **New Environment**.
+
+![Chaos Environments](./static/connect-chaos-infrastructures/chaos-environments.png)
+
+In the **Create a new environment** modal, provide a name for the environment, optionally you can also add a description and tags. Then select an environment type i.e. **Production** or **Non-Production**. Then select **Create**. This will instantly create a new environment.
+
+![New Environment](./static/connect-chaos-infrastructures/new-environment.png)
+
+## Add Chaos Infrastructure
+Now we can add a chaos infrastructure to our environment. Select **New Chaos Infrastructure**.
+
+You can either setup a chaos infrastructure on existing infrastructures that uses a Harness cluster connector i.e. Harness Kubernetes delegate or install the chaos infrastructure in a new infrastructure.
+
+To use an existing Harness Kubernetes delegate to install your chaos infrastructure, select **On Existing Infrastructures** in the modal. You can use any delegate under the Project, Organization, and Account scope.
+
+![Chaos Infrastructure in Existing Infra](./static/connect-chaos-infrastructures/chaos-infrastructure-in-existing-infra.png)
+
+:::info
+1. A Harness Kubernetes delegate with cluster-wide read/write access can setup both cluster-scoped and namespaced-scoped chaos infrastructures.
+2. A Harness Kubernetes delegate with cluster-wide read access **can't** be used to setup a chaos infrastructures.
+3. A Harness Kubernetes delegate with specific namespace access can only be used to setup a namespace-scoped chaos infrastructure in that specific Kubernetes namespace.
```

Comment:
I think here we may be referring to harness Kubernetes connectors with their specific harness Kubernetes Delegates configured to the particular settings we are describing here, ignore if this was not the intention